app/modules/import-export-customization/assets/js/shared/components/kit-content-customization-dialog.js

```diff
@@ -73,6 +75,11 @@ export function KitContentCustomizationDialog( {
 			handleSaveChanges={ () => handleSaveChanges( 'content', settings, true, unselectedValues.current ) }
 		>
 			<Stack>
+				{ isOldElementorVersion && (
+					<Box sx={ { mb: 2 } }>
```

Comment:
Add spacing to the Stack instead.
```suggestion
					<Box>
```

Reply:
but them spacing will be applied to all elements. is it expected?